src/TesApi.Web/AzureProxy.cs

```diff
@@ -40,6 +42,9 @@ public class AzureProxy : IAzureProxy
         private const string DefaultAzureBillingRegionName = "US West";
 
         private static readonly HttpClient httpClient = new();
+        private static readonly RetryPolicy batchRaceConditionJobNotFoundRetryPolicy = Policy
+            .Handle<BatchException>(ex => ex.RequestInformation.BatchError.Code == BatchErrorCodeStrings.JobNotFound)
+            .WaitAndRetry(10, retryAttempt => TimeSpan.FromSeconds(1));
```

Comment:
This should be WaitAndRetryAsync. The RetryPolicy type changes to AsyncRetryPolicy.